presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixCaching.java

```diff
@@ -498,10 +504,13 @@ public void testFileSystemBindings()
             assertRawFileSystemInstanceOf(fileSystem, CachingPrestoSecureAzureBlobFileSystem.class);
```

Comment:
update todo above to reference `https://github.com/prestosql/presto/issues/2380` by the way